development/build/utils.js

```diff
@@ -70,10 +70,38 @@ function getBrowserVersionMap(platforms, version) {
     const versionParts = [major, minor, patch];
     const browserSpecificVersion = {};
     if (prerelease) {
-      if (platform === 'firefox') {
-        versionParts[2] = `${versionParts[2]}${buildType}${buildVersion}`;
-      } else {
-        versionParts.push(buildVersion);
+      const { id } = loadBuildTypesConfig().buildTypes[buildType];
+      if (id < 10 || id > 64 || buildVersion < 0 || buildVersion > 999) {
```

Comment:
The `id` verification here is also down in the `build.yml` validation script, so maybe this is superfluous?